### DISCUSSION TITLE
Migration

### DISCUSSION THREAD

**Discussion by Dan-613** [jellyfin/jellyfin#12966] [Q&A] [answered]
Please forgive me for asking. I'm new to the opensource world but been developing for over 20 years.

Someone mention here that a migration would have to be created.
https://github.com/jellyfin/jellyfin-web/issues/6253

Does that require an issue to be created in this server repo before a migration can be created and submitted?

**Comment by Shadowghost** [comment, marked as answer]
You can submit PRs without there being an issue for it. Just put your reasoning into the PR description (in this case linking the web PR is sufficient).

**Comment by Dan-613** [comment]
Thanks for clarifying how to handle code changes for issues that affect multiple repos.
I've opened a new discussion regarding the topic of that bug report specifically.
